nxrm-ha/values.yaml

```diff
@@ -1,8 +1,8 @@
 # Declare variables to be passed into your templates.
 namespaces:
   nexusNs:
-    enabled: true
-    name: "nexusrepo"
+    enabled: false
+    name: "" # defaults to the Release Namespace
```

Comment:
@renesch-de Thank you for providing this capability. Please we'd prefer the default behaviour to be:
`enabled: true`
`name: "nexusrepo"`

This is because there could be users using it as is

Reply:
@bobotimi I understand your reasoning. I changed the implementation accordingly.